test/610_proxy_wait_for_weave_test.sh

```diff
@@ -31,7 +31,7 @@ assert_raises "proxy docker_on $HOST1 run -e 'WEAVE_CIDR=10.2.1.2/24' $BASE_IMAG
 # Check only user-specified volumes and /w are mounted
 proxy docker_on $HOST1 create --name c4 -v /tmp/1:/srv1 -v /tmp/2:/srv2 -e 'WEAVE_CIDR=10.2.1.3/24' $BASE_IMAGE $CHECK_ETHWE_UP
 proxy docker_on $HOST1 start -ai c4
-assert "docker_on $HOST1 inspect --format='{{range \$k, \$v := .Volumes}}{{\$k}} {{end}}' c4" "/srv1 /srv2 /w "
+assert "docker_on $HOST1 inspect --format='{{range .Mounts}}{{.Destination}} {{end}}' c4" "/srv1 /srv2 /w "
```

Comment:
isn't `Mounts` new in 1.8? i.e. are we switching from this test breaking on 1.8 to this test breaking on everything *except* 1.8?